packages/cli-repl/src/clr.ts

```diff
@@ -1,5 +1,8 @@
 import ansi from 'ansi-escape-sequences';
 
-export default function clr(text: string, style: any): string {
-  return process.stdout.isTTY ? ansi.format(text, style) : text;
+export default function clr(text: string, style: string|string[], options: { colors: boolean }): string {
```

Comment:
Wouldn't be a bit confusing without context for a `clr` function (that i believe stands for `color`) to take a `color` `true | false` argument? 

Should we rename something, or do something to make this clear?

Reply:
I would absolutely be in favor of renaming `clr` to something more expressive :) I’ll try to think of something

Reply:
I’ve renamed the functions themselves but not the places where they were imported, still not ideal but at least not that strongly abbreviated